krizten/Memoire-WebClient
# #165524917 Develop Login Page UI

### What does this PR do?
- Build out the user interface for the login page and "forgot password" modal .
#### Description of Task to be completed?
- Convert the UI/UX design provided to markup.
- Ensure the login page is responsive to screen sizes and matches the design provided.
#### How should this be manually tested?
- Clone this repository and checkout to the branch: ft/165524917/login-page
- Ensure [yarn](https://yarnpkg.com/lang/en/) is installed, then install the project dependencies by simply running ```yarn``` in the terminal.
- Start the project by running ```yarn start```.
- Using a web browser, navigate to ```http://localhost:3000/login``` to see the login page.
- On the login page, click "forgot password" to see the "forgot password" modal.
#### What are the relevant pivotal tracker stories?
#165524917
#### Screenshots (if appropriate)
![memoire-login-page-desktop](https://user-images.githubusercontent.com/29978455/57353587-e25c2700-7160-11e9-8cb8-930fbca19049.png)
![memoire-forgot-password-desktop](https://user-images.githubusercontent.com/29978455/57353599-ec7e2580-7160-11e9-876a-7e39a6ab2158.png)
![memoire-login-page-mobile](https://user-images.githubusercontent.com/29978455/57353604-f2740680-7160-11e9-8a91-da03f1e57386.png)
![memoire-forgot-password-mobile](https://user-images.githubusercontent.com/29978455/57353612-f738ba80-7160-11e9-9cb8-826536fe17bc.png)
